WooCommerce/Classes/Extensions/UIView+Helpers.swift

```diff
@@ -18,6 +18,13 @@ extension UIView {
     class func instantiateFromNib<T>() -> T {
         return loadNib().instantiate(withOwner: nil, options: nil).first as! T
     }
+
+    /// Returns whether there is a Nib associated with the receiver, being the filename its Class Name.
+    /// Use it to avoid *Could not load NIB in bundle* crash before calling `registerNib` if you are not sure that it exists.
+    ///
+    class func nibExistsInMainBundle() -> Bool {
```

Comment:
Yeah, I also considered that, but I went for  a method because the body is not so straightforward (`path(forResource:ofType:inDirectory:)` is not O(1) 😅